albinos-editor/docs/technical_doc_editor.md

```diff
@@ -0,0 +1,43 @@
+# albinos_editor
+
+## Prerequisites
+
+Below is the list of prerequisites to compile albinos-editor on your machine:
+
+* Nim version 0.18 minimum
+* Nimble
```

Comment:
Maybe add instructions on how to get this. Isn't as straightforward as you make it out to be.
This is mostly because as far I know most package managers don't provide nimble.
A simple link to the official download or something like that would suffice.